selfdrive/hardware/tici/hardware.h

```diff
@@ -25,7 +25,13 @@ class HardwareTici : public HardwareNone {
       brightness_control.close();
     }
   };
-  static void set_display_power(bool on) {};
+  static void set_display_power(bool on) {
+    std::ofstream bl_power_control("/sys/class/backlight/panel0-backlight/bl_power");
+    if (bl_power_control.is_open()) {
+      bl_power_control << (on ? "0" : "1") << "\n";
```

Comment:
FB_BLANK_POWERDOWN = 4
```suggestion
      bl_power_control << (on ? "0" : "4") << "\n";
```